src/Tables/molecules/TypeTable.tsx

```diff
@@ -80,10 +106,21 @@ const TypeTable : React.FC<IProps> = ({
           {hasStatus ? <HeaderItem fixedWidth={10} /> : null}
           {hasThumbnail ? <HeaderItem fixedWidth={70} /> : null}
           {hasTypeIcon ? <HeaderItem fixedWidth={35} /> : null}
-
           {columnConfig.map((column, key) => {
-            const {alignment, header, hasCopyButton} = column;
-            return <HeaderItem key={key} alignment={alignment} hasCopyButton={hasCopyButton}>{header}</HeaderItem>;
+            const {alignment, hasCopyButton, sortActive, ascending } : ITableColumnConfig = column;
+            return <HeaderItem
+                      key={key}
+                      alignment={alignment}
+                      hasCopyButton={hasCopyButton}
+                      >
+                        <TableHeaderTitle
+                          {...column}
+                          columnKey={key}
```

Comment:
Can we rename this to `columnId` to match the other interface?

Reply:
I talked about it in the code notes I left.
Since you mention that the UI shouldn't rely on getting the columnId. I'm actually not using it.
This is the indexKey.

I could rename it to indexKey to avoid confusion.